docs/spec-files/message_structure.md

```diff
@@ -75,37 +75,36 @@ The following example shows common elements of a plaintext message. Further deta
 
 ### Message Headers
 
-A DIDComm plaintext message conveys most of its application-level data inside a JSON `body` object that is a direct child of the message root. The structure inside `body` is predicted by the value of the message's `type` attribute, and varies according to the definition of the protocol-specific message in question. Each `type` of message will have its own `body`. 
+A **DIDComm plaintext message** conveys most of its application-level data inside a JSON `body` object that is a direct child of the message root. The structure inside `body` is predicted by the value of the message's `type` attribute, and varies according to the definition of the protocol-specific message in question. Each `type` of message will have its own `body`. 
 
 However, some attributes are common to many different message types. When metadata about a message means the same thing regardless of context, and when it is susceptible to generic rather than message-specific handling, that metadata can be placed in **headers**. Headers are siblings of `body` and may be added to any message type. They are encrypted and decrypted along with `body` and therefore have an identical audience.
 
-Headers in DIDComm are intended to be extensible in much the same way that headers in HTTP or SMTP are extensible. A few headers are predefined:
+Headers in [DIDComm Messaging](.) are intended to be extensible in much the same way that headers in HTTP or SMTP are extensible. A few headers are predefined:
 
 - **id** - REQUIRED. Message ID. The `id` attribute value MUST be unique to the sender.
 
-- **type** - REQUIRED. Plaintext message type, useful for message handling in application-level protocols. The `type` attribute value MUST be a valid [Message Type URI](protocols.md#message-type-uri), that when resolved gives human readable information about the message. The attribute's value predicts the content in the `body` of the message, for example the presence of other attributes and how they should be processed.
+- **type** - REQUIRED. Plaintext message type, useful for message handling in application-level protocols. The `type` attribute value MUST be a valid [Message Type URI](protocols.md#message-type-uri), that when resolved gives human readable information about the message. The attribute's value SHOULD predict the content in the `body` of the message.
 
-- **typ** - OPTIONAL. Media type of the JWM content. MUST be set to `application/didcomm-plain+json` for **DIDComm plaintext messages**. This tells generic JOSE libraries that the JWM in question is DIDComm rather than a different JWM type and is therefore helpful if messages will be processed in a system that supports a variety of JWMs.
+- **typ** - OPTIONAL. Media type of the JWM content.
```

Comment:
@kdenhartog could you pls check whether you are ok with the current language. I removed redundant normative statement on `typ` MUST be `application/didcomm-plain+json`.